sensor_msgs/Image im
int64 cameraFrameId
int8 howManyBoxes
yolorosort/singleBBox[] boxesWithAll
float32[] sortedBoxes
time stampOfOriginal
